### PR TITLE
chore: depricate mkdocs and outdated docs

### PR DESCRIPTION
**Pull request by katrilh**
## Description
Removing references to everything it  mkdocs  related, and the build action. 

The pre-commit ran for the tutorials after they were moved. no functional changes 

Still outstanding: How to remove the site that has the outdated documentation 

## Checklist:
- [ ] Tests added/updated.
- [ ] Documentation updated.
- [ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/power-ops-sdk/blob/master/CHANGELOG.md).
- [ ] Version bumped. If triggering a new release is desired, bump the version number in [pyproject.toml](https://github.com/cognitedata/power-ops-sdk/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -16,9 +16,11 @@ repos:
         args:
           - --exit-non-zero-on-fix
           - --config=pyproject.toml
+        exclude: "tutorials/"
       - id: ruff-format
         args:
           - --config=pyproject.toml
+        exclude: "tutorials/"
```
Comment (nodegard) [inline .pre-commit-config.yaml:23]: **suggestion:** better to add this excludes to the ruff definition in `pyproject.toml` so the configuration will apply if someone just runs ruff directly from the cli too

File: pyproject.toml [hunk 2]
```diff
@@ -29,21 +29,9 @@ StrEnum = {version = "^0.4.15", python = "<3.11"}
...
-mkdocs-git-authors-plugin = {version="*", optional=true}
-mkdocs-gitbook = {version="*", optional=true}
-mkdocs-glightbox = {version="*", optional=true}
 pymdown-extensions = {version="*", optional=true}
-mkdocstrings = {version="*", optional=true, extras=["python"]}
 cachelib = "^0.13.0"
 
-[tool.poetry.extras]
-docs = ["mkdocs", "mkdocs-jupyter", "mkdocs-material", "mkdocs-material-extensions", "mkdocs-git-revision-date-localized-plugin",
-    "mkdocs-git-authors-plugin", "mkdocs-gitbook", "mkdocs-glightbox", "pymdown-extensions", "mkdocstrings"]
```
Comment (nodegard) [inline pyproject.toml:46]: 🫶🏻

File: .gitignore [hunk 3]
```diff
@@ -33,6 +33,9 @@ test.py
 do_release.py
 *.env*
 
+# mk docs build
+site/
+
```
Comment (nodegard) [inline .gitignore:38]: **question:** why is this needed since we removed mk docs?

Comment (katrilh) [reply .gitignore:38]: Just missed it when in my search to remove, it's not needed

**Review by nodegard — APPROVED** [review]
